0.notes.md

```diff
@@ -0,0 +1,1220 @@
+            ],
+            "allow_restricted_indices": false
+          },
+          {
+            "names": [
+              "logs-elastic_agent.packetbeat-default"
+            ],
+            "privileges": [
+              "auto_configure",
+              "create_doc"
+            ],
+            "allow_restricted_indices": false
+          },
+          {
+            "names": [
+              "metrics-elastic_agent.packetbeat-default"
+            ],
+            "privileges": [
+              "auto_configure",
+              "create_doc"
+            ],
+            "allow_restricted_indices": false
+          }
+        ],
+        "applications": [],
+        "run_as": [],
+        "metadata": {},
+        "type": "role"
+      }
+    },
+    "limited_by_role_descriptors": {
+      "elastic/fleet-server": {
+        "cluster": [
+          "monitor",
+          "manage_own_api_key"
+        ],
+        "indices": [
+          {
+            "names": [
+              "logs-*",
+              "metrics-*",
+              "traces-*",
+              "synthetics-*",
+              ".logs-endpoint.diagnostic.collection-*",
+              ".logs-endpoint.action.responses-*"
+            ],
+            "privileges": [
+              "write",
+              "create_index",
+              "auto_configure"
+            ],
+            "allow_restricted_indices": false
+          },
+          {
+            "names": [
+              "traces-apm.sampled-*"
+            ],
+            "privileges": [
+              "read",
+              "monitor",
+              "maintenance"
+            ],
+            "allow_restricted_indices": false
+          },
+          {
+            "names": [
+              ".fleet-*"
+            ],
+            "privileges": [
+              "read",
+              "write",
+              "monitor",
+              "create_index",
+              "auto_configure",
+              "maintenance"
+            ],
+            "allow_restricted_indices": true
+          }
+        ],
+        "applications": [
+          {
+            "application": "kibana-*",
+            "privileges": [
+              "reserved_fleet-setup"
+            ],
+            "resources": [
+              "*"
+            ]
+          }
+        ],
+        "run_as": [],
+        "metadata": {},
+        "type": "role"
+      }
+    },
+    "name": "b91f9075-5f1b-45f1-949c-d7b852e88b7a:default",
+    "version": 8030199,
+    "metadata_flattened": {
+      "agent_id": "b91f9075-5f1b-45f1-949c-d7b852e88b7a",
+      "managed_by": "fleet-server",
+      "managed": true,
+      "type": "output"
+    },
+    "creator": {
+      "principal": "elastic/fleet-server",
+      "full_name": "Service account - elastic/fleet-server",
+      "email": null,
+      "metadata": {
+        "_elastic_service_account": true
+      },
+      "realm": "_service_account",
+      "realm_type": "_service_account"
+    }
+  }
+}
+```
+
+</details>
```

Comment:
This shouldn't be part of this PR?

Reply:
it didn't fit on the PR description. Too long :/